### PR TITLE
version up modules.

### PR DESCRIPTION
**Pull request by lancard**
version up modules.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Spring Boot Starter Handlebars
 ====
 
-[![Build Status](https://travis-ci.org/allegro/handlebars-spring-boot-starter.svg?branch=master)](https://travis-ci.org/allegro/handlebars-spring-boot-starter)
```
Comment (platan) [inline README.md]: Could you please revert it to `https://travis-ci.org/allegro/...`?

File: test-dependencies/src/test/groovy/pl/allegro/tech/boot/autoconfigure/handlebars/HandlebarsHelpersDependenciesSpec.groovy [hunk 2]
```diff
@@ -5,15 +5,13 @@ import org.springframework.mock.web.MockServletContext
...
-import static org.springframework.boot.test.util.EnvironmentTestUtils.addEnvironment
-
 class HandlebarsHelpersDependenciesSpec extends Specification {
 
     def context = new AnnotationConfigWebApplicationContext()
 
     def setup() {
         context.servletContext = new MockServletContext()
-        addEnvironment(context)
+        TestPropertyValues.empty().applyTo(context);
```
Comment (platan) [inline test-dependencies/src/test/groovy/pl/allegro/tech/boot/autoconfigure/handlebars/HandlebarsHelpersDependenciesSpec.groovy]: This `;` is redundant. I suggest to removed it.

**Review by platan — CHANGES_REQUESTED** [review]
@lancard Thanks again for this change. I've left two comments.

File: README.md (hunk 1 above)
Comment (lancard) [reply README.md]: reverted

**Review by platan — APPROVED** [review]
👍 @lancard

**Comment by platan** [comment]
@plesiecki Can you look at this PR?

**Review by plesiecki — APPROVED** [review]

**Comment by platan** [comment]
I will release a new version and I will let you know when the new version is available.

**Comment by platan** [comment]
@lancard version 0.3.2 is available.

**Pull request merged**